sysl2/sysl/Parser_test.go

```diff
@@ -111,171 +154,181 @@ func parseAndPrint(t *testing.T, filename, root string) error {
 	return nil
 }
 
-func testParse(t *testing.T, filename, root string) {
-	equal, err := parseAndCompare(filename, root)
+func testParseAgainstPython(t *testing.T, filename, root string) {
```

Comment:
root is only non-empty in two of the tests below (lines 263 and 329). In all other cases, the filename is given with the path. Is root something we want to keep for future or do we get want to get rid of it?

Reply:
root is required for import statements to work.